service/frontend/dcRedirectionHandler.go

```diff
@@ -1671,32 +1704,132 @@ func (handler *DCRedirectionHandlerImpl) UpdateWorkflow(
 	return resp, err
 }
 
+func (handler *DCRedirectionHandlerImpl) StartBatchOperation(
```

Comment:
did you mess up your commit history or do you have to rebase to latest master? This should no appear in your PR.